packages/cli/src/services/project-initializer.ts

```diff
@@ -19,7 +19,7 @@ export async function generateConfigFiles(config: ProjectInitializerConfig): Pro
 
 export async function installDependencies(config: ProjectInitializerConfig): Promise<void> {
   try {
-    await exec('npm install', { cwd: projectDir(config) })
+    await exec('npx yarn install', { cwd: projectDir(config) })
```

Comment:
Maybe we should move this method to `/services/dependencies`

Reply:
refactored in 603b79440028bcf68b3e1ba82b07ec75e3dea108